docs/sources/panels/transformations.md

```diff
@@ -259,6 +259,41 @@ Here is the result after applying the `Series to rows` transformation.
 | 2020-07-07 09:30:57 | Humidity    | 33    |
 | 2020-07-07 09:30:05 | Temperature | 19    |
 
+
+## Filter By Value
+
+This transformation allows you to filter your data directly in Grafana. This allows you to remove some of the data points from your query result. You can either include or exclude some of your data points (rows) from a field that match the filter test (`Equal`, `Greater`, `Lower`, `isNull`, `Regex`, `Equal` and a few more).
+
+Consider the following data set:
+
+| Time                | Temperature |
+|---------------------|--------------|
+| 2020-07-07 11:34:23 | 32    |
+| 2020-07-07 11:34:22 | 28    |
+| 2020-07-07 11:34:21 | 26   |
+| 2020-07-07 11:34:20 | 23    |
+| 2020-07-07 10:32:24 | 31    |
+| 2020-07-07 10:31:22 | 20    |
+| 2020-07-07 09:30:57 | 19    |
+
+For instance, you can `Include` all data points that have a **Temperature** lower than 30°C, you would get the following result :
```

Comment:
```suggestion
For instance, you can `Include` all data points that have a **Temperature** lower than 30°C, you would get the following result:
```

Reply:
What I intend to do is to make simple sentences that translate to a configuration in the UI:

> You can `Include` all data points that have a **Temperature** lower than 30°C, you would get the following result

This translates to `Include` `Temperature` `Lower Than` `30` in the UI.

Maybe I should put all the parts in inline code `like this` instead, or all in bold (but bold you won't see the separation between `temperature` and `lower than`.

Reply:
I like what you're doing, but I skew towards less formatting rather than more. In this example, I would say, "...you can **Include** all data points that have a temperature lower than 30°C. You would get the following result: